bot_script.txt

```diff
@@ -1,27 +1,27 @@
-Hello, I am Chorus! We'll be getting started in just a moment!
-
-We want to give everyone a little time to get connected to the stream.
+======
 
-This is Chorus, your host and party rock commander. Let's give everyone just a few more moments.
 
-I think it's just about time, don't you?
+Welcome to the listening party for One Hour Compo, Round 425! I'm your host, Chorus. We'll be getting the party started in just a few moments. :chorusgreet: 
```

Comment:
Hmm, this is the script for the TTS, not the text that Chorus posts to the discord, tho.

So we need sections for the static scripts I'll generate once and put together with music (intro, and maybe outro if you want that), and then dynamic scripts with probably a few options (first track, random track).

Reply:
You can PR the code directly to update what the discord bot says in chat in this PR as well: https://github.com/fusion2004/chorus/blob/master/scripts/stream.js#L54

Basically, all of the `res.send` lines.

Reply:
They moved to the commands/ directory after the Discord.js conversion, btw.